docs/designers-developers/developers/tutorials/block-based-themes/README.md

```diff
@@ -0,0 +1,375 @@
+# Creating a block-based theme
+
+The purpose of this tutorial is to show how to create a basic block based theme
+and help theme developers transition to full site editing.
+
+You will learn about the required files, how to combine templates and template parts,
+how to add presets for global styles, and how to add blocks and export the templates in the site editor.
+
+Full site editing is an experimental feature and the workflow in this tutorial is likely to change.
+This tutorial was written for Gutenberg version 8.5.
+
+Additional documentation for block-based themes is available at https://developer.wordpress.org/block-editor/developers/themes/block-based-themes/
+
+## Table of Contents
+ 1. [What is needed to create a block-based theme?](/docs/designers-developers/developers/tutorials/block-based-themes/README.md#what-is-needed-to-create-a-block-based-theme)
+ 2. [Creating the theme](/docs/designers-developers/developers/tutorials/block-based-themes/README.md#creating-the-theme)
+ 3. [Creating the templates and template parts](/docs/designers-developers/developers/tutorials/block-based-themes/README.md#creating-the-templates-and-template-parts)
+ 4. [Experimental-theme.json -Global styles](/docs/designers-developers/developers/tutorials/block-based-themes/README.md##experimental-themejson--global-styles)
+ 5. [Adding blocks](/docs/designers-developers/developers/tutorials/block-based-themes/adding-blocks.md)
+
+## What is needed to create a block-based theme?
+
+To use a block based theme you need to have Gutenberg installed and full site editing must be enabled.
+Full site editing can be enabled from the Gutenberg experiments menu in the WordPress admin area.
+
+A block-based theme is built using HTML templates and template parts.
+Templates are the main files used in the [template hierarchy](https://developer.wordpress.org/themes/basics/template-hierarchy/), 
+for example index, single or archive.
+Templates can optionally include structural template parts, for example a header, footer or sidebar.
+
+Each template or template part contains the [block grammar](https://developer.wordpress.org/block-editor/principles/key-concepts/#blocks), the HTML, for the selected blocks.
+The block HTML is generated in and exported from the **site editor**. It can also be added to the themes HTML files manually.
```

Comment:
```suggestion
The block HTML is generated in and exported from the **site editor**. It can also be added to the theme's HTML files manually.
```

Minor — missing an apostrophe here.